cohesive_energy.py

```diff
@@ -0,0 +1,28 @@
+from ase.md.velocitydistribution import (MaxwellBoltzmannDistribution,Stationary,ZeroRotation)
+from ase.md.verlet import VelocityVerlet
+from asap3 import Trajectory
```

Comment:
This should use Trajetory from ase.io to be able to be run in parallell.

Reply:
Fixed!

Reply:
Have these changes been pushed? I can't see them

Reply:
Nope, still resolving the other comments